### PR TITLE
Support filter drawers

### PR DESCRIPTION
**Pull request by maxgerber**

### CODE REVIEW

File: components/SupportFilter.js [hunk 1]
```diff
@@ -48,13 +52,22 @@ const getNewFilters = ({ checked, name, checkedFilters }) => {
 }
 
 const SortButton = styled.div.attrs(({ sortTypeSelected }) => ({
-  className: `py-1.75 px-6 w-1/3 text-center border border-solid border-lightgray rounded-2.5 ${
+  className: `py-1.75 px-6 w-1/3 text-center border-r last:border-none border-solid border-midgray ${
```
Comment (lucymk) [inline components/SupportFilter.js:55]: that's cool I didn't know you could do last: things in tailwind

**Review by lucymk — APPROVED** [review]

File: components/SupportFilter.js [hunk 2]
```diff
@@ -131,4 +140,58 @@ const SupportFilter = ({ filters, sort, applyFiltersAndSort }) => {
...
+  }
+
+  return (
+    <Fragment key={`filter-cat-${index}`}>
+      <CategoryButton onClick={toggleCategory}>
+        <div>{title}</div>
+        <img
+          src="/icons/drawer.svg"
+          alt="Drawer closed"
+          className={isOpen ? '' : 'transform rotate-180'}
```
Comment (lucymk) [inline components/SupportFilter.js:172]: I think they're round the wrong way!

File: pages/support/[category]/index.js [hunk 3]
```diff
@@ -91,6 +91,12 @@ const Category = ({ supportServices, query: { category } }) => {
   return (
     <>
       <CategoryStyled>
+        <button
```
Comment (lucymk) [inline pages/support/[category]/index.js:94]: This button replaces the one in the navbar

**Review by lucymk — CHANGES_REQUESTED** [review]

File: components/SupportFilter.js (hunk 2 above)
Comment (maxgerber) [reply components/SupportFilter.js:172]: they are !

Comment (maxgerber) [reply components/SupportFilter.js:172]: my bad

File: components/SupportFilter.js (hunk 1 above)
Comment (maxgerber) [reply components/SupportFilter.js:55]: pretty good huh https://tailwindcss.com/docs/configuring-variants

**Review by lucymk — APPROVED** [review]

**Pull request merged**